templates/cassiopeia/index.php

```diff
@@ -115,6 +115,13 @@
 
 // Container
 $wrapper = $this->params->get('fluidContainer') ? 'wrapper-fluid' : 'wrapper-static';
+$navPadding = $this->params->get('removepaddingHorizontalnav', 0);
+
+if ($navPadding) {
+    $navPadding = 'remove-padding-nav';
```

Comment:
```suggestion
    $navPadding = ' remove-padding-nav';
```
Add the space here instead of line 152 so we do not have an unnecessary space when `$navPadding` was set to an empty string in line 123.

Reply:
I can see this suggestion was applied, but my other one below also needs to be applied.